### PR TITLE
Update Ember-CLI to 1.13.13

### PR DESCRIPTION
**Pull request by acburdine**
no issue

TODO:
- [x] update ember-cli
- ~~ic-ajax => ember-ajax~~ (should be a separate PR)

### CODE REVIEW

**Comment by acburdine** [comment]
@kevinansfield I looked into switching ic-ajax to ember-ajax, and I got a good ways into it, but it also involves a lot more changes than I think are in the scope of this PR. I can work on finishing up the switch and just submit another PR.

**Comment by kevinansfield** [comment]
:+1:  Agreed, the `ember-ajax` switch can definitely be a separate PR.

Can you remove the `crossdomain.xml` and `robots.txt` files? They're not used at all when the admin app is served through the ghost process so it's probably confusing to keep them around.

**Comment by kevinansfield** [comment]
Also, this will want bumping to `1.13.13` :wink: I think it's only `loader.js` and `ember-cli-sri` version bumps.

File: core/client/bower.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "jqueryui-touch-punch": "furf/jquery-ui-touch-punch",
     "jquery.simulate.drag-sortable": "0.1.0",
     "keymaster": "1.6.3",
-    "loader.js": "3.2.1",
+    "loader.js": "3.3.0",
```
Comment (kevinansfield) [inline core/client/bower.json]: Is there are particular reason we're not specifying this the same way as ember-cli? `"loader.js": "ember-cli/loader.js#3.4.0",` - there must be a reason why they're explicit about which `loader.js` to pull in.

File: core/client/bower.json [hunk 2]
```diff
@@ -5,12 +5,12 @@
     "codemirror": "5.2.0",
     "devicejs": "0.2.7",
     "ember": "1.13.10",
-    "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.3",
-    "ember-cli-test-loader": "0.1.3",
-    "ember-data": "1.13.13",
+    "ember-cli-shims": "0.0.6",
+    "ember-cli-test-loader": "0.2.1",
+    "ember-data": "1.13.15",
```
Comment (kevinansfield) [inline core/client/bower.json]: I think we should be making `ember-data` updates in a separate PR so that it's easier to track down any behaviour changes.

**Comment by kevinansfield** [comment]
Quick ping - would you mind rebasing this when you get a moment? Excited to get this merged and the resulting super fast rebuilds :grinning:

**Pull request merged**